app/src/main/java/com/project/gamedb/ui/ranking/RankingViewHolder.kt

```diff
@@ -0,0 +1,43 @@
+package com.project.gamedb.ui.ranking
+
+import android.view.View
+import android.widget.ImageView
+import com.project.gamedb.base.BaseViewHolder
+import com.project.gamedb.data.model.Games
+import com.project.gamedb.ultis.*
+import kotlinx.android.synthetic.main.item_recyclerview_ranking.view.*
+
+class RankingViewHolder(private val itemView: View) : BaseViewHolder<Games>(itemView) {
+    private var itemFeature: Games? = null
+
+    override fun onBind(item: Games) {
+        itemFeature = item
+        itemView.run {
+            textRankingGame.text = item.gameName
+            imageRanking.loadImage(item.gameBackgroundImage)
+            val platform = getPlatform(item.gamePlatforms.toString())
+            var size = platform.size
+            val imageRanking = listOf<ImageView>(
+                image1stRankingPlatform,
+                image2ndRankingPlatform,
+                image3rdRankingPlatform,
+                image4thRankingPlatform,
+                image5thRankingPlatform
+            )
+            for (image in imageRanking) {
+                if (size > 0) {
+                    image.setPlatform(platform[size - 1])
+                    --size
+                } else break
+            }
```

Comment:
Đoạn này logic là gì nhỉ, mục đích làm gì thế ?

Reply:
Bài toán là có 5 slot imageView trống được sắp xếp cạnh nhau, list nhận về có thể có ít hơn 5 url, vì vậy phải sắp xếp từng cái phần tử đứng cạnh nhau lần lượt, những cái imageView phía sau tiếp tục để trống
Kiểu như là : 11100 chứ không phải 10110

Reply:
sao anh không reverse list lại rồi dùng for cho đơn giản nhỉ?